### PR TITLE
Added french locales

### PR DESCRIPTION
**Pull request by SebastienFCT**
~~Note that while french locales are expecting %d for 0 and 1, in this context it is unnecessary (and wouldn't make sense).~~

~~Android Studio throws a warning but it doesn't prevent the developer from compiling. I'm not a fan of adding a silent ignore as they are usually easy to miss on code review.~~

EDIT: The CI tool rejected the pull request for the reason mentioned above so I modified the strings to support the placeholder in the singular form

### CODE REVIEW

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<!--
+  ~ Copyright 2018 Google LLC
```
Comment (tiembo) [inline app/src/main/res/values-fr/strings.xml]: Can you update the copyright year to 2019?

File: app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<resources>
+
```
Comment (tiembo) [inline app/src/main/res/values-fr/strings.xml]: Lines 18-23 can be removed

File: app/src/main/res/values-fr/strings.xml (hunk 1 above)
Comment (SebastienFCT) [reply app/src/main/res/values-fr/strings.xml]: Done

File: app/src/main/res/values-fr/strings.xml (hunk 2 above)
Comment (SebastienFCT) [reply app/src/main/res/values-fr/strings.xml]: Done, I assumed that your referred to non-translatable rows so I also removed the line 48. I compared the final result with other existing locales

File: app/src/main/res/values-fr/strings.xml [hunk 3]
```diff
@@ -0,0 +1,46 @@
+<!--
+  ~ Copyright 2019 Google LLC
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (dazza5000) [inline app/src/main/res/values-fr/strings.xml]: people typically use * for the line indicators in the copyright

File: app/src/main/res/values-fr/strings.xml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<resources>
+    <string name="menu_filter_by_grow_zone">Filtrer par zone de croissance</string>
+    <string name="my_garden_title">Mon jardin</string>
+    <string name="plant_detail_image_content_description">Image de plante</string>
+    <string name="plant_list_title">Liste de plantes</string>
```
Comment (yrezgui) [inline app/src/main/res/values-fr/strings.xml]: Liste de**s** plantes

File: app/src/main/res/values-fr/strings.xml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    <string name="plant_details_title">Détails sur la plante</string>
+    <string name="navigation_drawer_header_content_description">Logo de l\'application du volet de navigation</string>
+    <string name="added_plant_to_garden">Plante ajoutée à votre jardin</string>
+    <string name="garden_empty">Votre jardin est vide</string>
+    <string name="planted_date">%1$s planté(e) let %2$s</string>
```
Comment (yrezgui) [inline app/src/main/res/values-fr/strings.xml]: %1$s planté(e) le~~t~~ %2$s

**Comment by yrezgui** [comment]
I added typos fixes

**Comment by tiembo** [comment]
Thank you @SebastienFCT !

**Pull request merged**